src/notification/subscribe/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "subscribe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "subscribe.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "subscribe": "npx ts-node subscribe.ts"
```

Comment:
@KevinLemon112 The reason why the subscribe notification service wasn't working previously was there was no `npx ts-node` that is required to compile and run typescript code.